pkg/routes/routes_test.go

```diff
@@ -43,17 +45,19 @@ func TestPingdomEmpty(t *testing.T) {
 	assert.Nil(t, dmsRouter.Pingdom(c))
 	assert.Equal(t, http.StatusOK, rec.Code)
 
-	assert.Nil(t, etcD.Purge())
+	assert.Nil(t, etcD.Purge("env"))
 }
 
 func TestPingdomValid(t *testing.T) {
+	// launch etcd server
```

Comment:
Renaming `setup` to `startEtcd` would make the comment redundant. But that change can be done in separate PR.